steering_vectors/train_steering_vector.py

```diff
@@ -57,9 +57,13 @@ def train_steering_vector(
     pos_activations: dict[int, list[Tensor]] = defaultdict(list)
     neg_activations: dict[int, list[Tensor]] = defaultdict(list)
     # TODO: batching
-    for pos_prompt, neg_prompt in tqdm(
+    for pair in tqdm(
         training_samples, disable=not show_progress, desc="Training steering vector"
     ):
+        if isinstance(pair, SteeringVectorTrainingSample):
```

Comment:
Why was this if-statement added? `SteeringVectorTrainingSample` is already a tuple, so this should be unnecessary. If this is fixing a bug, please add an accompanying test case proving the fix

Reply:
My bad, I'll get rid of this

Reply:
fixed